docs/explore/introduction/documentation-feedback.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Feedback
+sidebarHeader: Explore
+sidebarSubHeader:
+pageHeader: Explore
+path: /explore/introduction/feedback.html
+outline: deep
+tags:
+---
+
+<PageHeader/>
+
+<SearchHighlight/>
+
+<FlexStartTag/>
+
+# {{$frontmatter.title}} to the API3 docs
+
+We always appreciate your thoughts and suggestions about our documentation.
+Whether it's concerning the presentation of the information, or even provided
+through our feedback template, we're eager to hear your feedback.
+
+### Feedback or sharing ideas
+
+If you are reading our documentation and have ideas on how we can better present
+or communicate these concepts we would like to hear from you.
+
+This might include scenarios such as:
+
+- Ideas for us to communicate technical ideas in a simpler way
+- Areas that you think we could emphasise the 'why' behind our technical design
+- Points of difference within API3 that you think we should further highlight or
+  communicate
+
+Please email ecosystem@api3.org with the subject "Documentation feedback
+submission".
+
+### Submit an issue
+
+We welcome all developers to contribute to the growth and enhancement of API3
+DAO by submitting any issues or suggestions you come across via our issue
+template. Your insight is invaluable in making our technology better.
+
+Please submit to https://github.com/api3dao/vitepress-docs/issues.
+
+::: tip Feedback is valueable
+
+Feedback is valueable and any submission shared with be rewarded with a retrospective bounty of up to $250 of API3 
+
+:::
+
+## Contributing to the Technical Documentation
+
+See the [dev documentation](/dev/) for an overview of the API3 technical
+documentation site. It contains instructions on the make-up of the docs and on
+how to clone, setup, and run the project locally. For non-DAO members a PR from
+a fork is always welcome.
+
+You can also see the [Contributing](/docs/explore/introduction/contributing.md) overview on how to engage further.
+
+### Join the API3 testers guild
+
+API3 Testers Guild is a community of professionals involved to testing API3
+products & services. The Guild often conducts workshops, webinars, and
+conferences to foster a dynamic and up-to-date understanding of testing in the
+evolving tech landscape.
```

Comment:
Are there any links or more info for joining this?

Reply:
There will be soon, it's not firmed up yet. It will be added to the ecosystem gitbook - can review and update here. 

Should I resolve this?

Reply:
Should we take this out until there is something concrete then?

Reply:
kk taken out